sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/implementation/servicebus/AzureServiceBusMessagingAutoConfiguration.java

```diff
@@ -80,8 +84,19 @@ static class ProcessorContainerConfiguration {
         @ConditionalOnMissingBean
         ServiceBusProcessorFactory defaultServiceBusNamespaceProcessorFactory(
             NamespaceProperties properties,
-            ObjectProvider<PropertiesSupplier<ConsumerIdentifier, ProcessorProperties>> suppliers) {
-            return new DefaultServiceBusNamespaceProcessorFactory(properties, suppliers.getIfAvailable());
+            ObjectProvider<PropertiesSupplier<ConsumerIdentifier, ProcessorProperties>> suppliers,
```

Comment:
What are these for?

Reply:
For Service Bus Integration scenario, it's used to custom the Service Bus builder factory in `DefaultServiceBusNamespaceProcessorFactory`.